src/esp/assets/ResourceManager.cpp

```diff
@@ -882,9 +883,24 @@ bool ResourceManager::loadInstanceMeshData(
 
     for (uint32_t iMesh = start; iMesh <= end; ++iMesh) {
       scene::SceneNode& node = parent->createChild();
-      node.addFeature<gfx::GenericDrawable>(
-          *meshes_[iMesh]->getMagnumGLMesh(), shaderManager_, NO_LIGHT_KEY,
-          PER_VERTEX_OBJECT_ID_MATERIAL_KEY, drawables);
+
+      gfx::Drawable::Flags meshAttributeFlags{};
+      if (meshes_[iMesh]->getMeshData()->hasAttribute(
+              Mn::Trade::MeshAttribute::Tangent)) {
+        meshAttributeFlags |= gfx::Drawable::Flag::HasTangent;
+      }
+      if (meshes_[iMesh]->getMeshData()->hasAttribute(
+              Mn::Trade::MeshAttribute::Bitangent)) {
+        meshAttributeFlags |= gfx::Drawable::Flag::HasBitangent;
```

Comment:
I'd name it `HasSeparateBitangent` to avoid confusion because four component tangents *technically* also have a bitangent.

Reply:
Makes sense. Will do.